packages/block-library/src/post-featured-image/edit.js

```diff
@@ -31,14 +53,44 @@ function PostFeaturedImageDisplay( {
 			featuredImage && select( 'core' ).getMedia( featuredImage ),
 		[ featuredImage ]
 	);
-	const image = ! media ? (
-		placeholderChip
-	) : (
-		<img
-			src={ media.source_url }
-			alt={ media.alt_text || __( 'No alternative text set' ) }
-		/>
-	);
+	const onSelectImage = ( value ) => {
+		if ( value?.id ) {
+			setFeaturedImage( value.id );
+		}
+	};
+	function onUploadError( message ) {
+		noticeOperations.removeAllNotices();
+		noticeOperations.createErrorNotice( message );
+	}
+	let image;
+	if ( ! featuredImage ) {
+		image = (
+			<MediaPlaceholder
+				icon={ <BlockIcon icon={ postFeaturedImage } /> }
+				onSelect={ onSelectImage }
+				notices={ noticeUI }
+				onError={ onUploadError }
+				accept="image/*"
+				allowedTypes={ ALLOWED_MEDIA_TYPES }
+				labels={ {
+					title: __( 'Featured image' ),
+					instructions: __(
+						'Upload a media file or pick one from your media library.'
+					),
+				} }
+			/>
+		);
+	} else {
+		// We have a Featured image so show a Placeholder if is loading.
+		image = ! media ? (
+			placeholderChip
+		) : (
+			<img
+				src={ media.source_url }
+				alt={ media.alt_text || __( 'No alternative text set' ) }
```

Comment:
`No alternative text set` - it doesn't seem like useful information for someone who can't see the image.

Reply:
Yeah, that was discussed in another PR when I had introduced that... No strong opinions here whether to remove it..

Reply:
How about "Featured image"? It should be what you would want to hear when you can't see the image.

Reply:
Sound good. I'll change it.